panel/widgets/__init__.py

```diff
@@ -34,6 +34,7 @@
 """
 from .base import CompositeWidget, Widget  # noqa
 from .button import Button, MenuButton, Toggle  # noqa
+from .chatbox import ChatBox  # noqa
```

Comment:
Are there any other places I need to add an import?

When I try to import it outside the widgets directory, I get:
```
  File "/Users/airbook/test.py", line 4, in <module>
    pn.widgets.ChatBox().servable()
AttributeError: module 'panel.widgets' has no attribute 'ChatBox'
```

Reply:
It seems to work fine for me:

![image](https://user-images.githubusercontent.com/19758978/235082067-7fd00757-11d9-4d9b-b418-d5b7460223fb.png)

Are you in the correct environment?

Reply:
Yeah; perhaps my environment is corrupted. I'll try recreating later.
```
(panel_dev) airbook@AHs-MacBook-Pro ~ % panel serve test.py
2023-04-28 09:50:39,763 Starting Bokeh server version 3.1.0 (running on Tornado 6.3)
2023-04-28 09:50:39,763 User authentication hooks NOT provided (default user enabled)
2023-04-28 09:50:39,765 Bokeh app running at: http://localhost:5006/test
2023-04-28 09:50:39,765 Starting Bokeh server with process id: 42681
2023-04-28 09:50:41,196 Error running application handler <bokeh.application.handlers.script.ScriptHandler object at 0x119d68ee0>: module 'panel.widgets' has no attribute 'ChatBox'
File 'test.py', line 4, in <module>:
pn.widgets.ChatBox().servable() Traceback (most recent call last):
  File "/Users/airbook/miniforge3/envs/panel_dev/lib/python3.9/site-packages/bokeh/application/handlers/code_runner.py", line 231, in run
    exec(self._code, module.__dict__)
  File "/Users/airbook/test.py", line 4, in <module>
    pn.widgets.ChatBox().servable()
AttributeError: module 'panel.widgets' has no attribute 'ChatBox'

2023-04-28 09:50:41,438 WebSocket connection opened
2023-04-28 09:50:41,439 ServerConnection created
2023-04-28 09:50:42,364 WebSocket connection closed: code=1001, reason=None
^C
Interrupted, shutting down
(panel_dev) airbook@AHs-MacBook-Pro ~ % which panel
/Users/airbook/miniforge3/envs/panel_dev/bin/panel
```